data/src/main/java/com/microsoft/azure/kusto/data/ClientRequestProperties.java

```diff
@@ -23,10 +29,11 @@
     private static final String PARAMETERS_KEY = "Parameters";
     public static final String OPTION_SERVER_TIMEOUT = "servertimeout";
     public static final String OPTION_CLIENT_REQUEST_ID = "ClientRequestId";
-    private static final long NANOS_TO_MILLIS = 1000000L;
-    private HashMap<String, Object> parameters;
-    private HashMap<String, Object> options;
-
+    private final HashMap<String, Object> parameters;
+    private final HashMap<String, Object> options;
```

Comment:
1) HashMap -> Map
2) This naming conflicts with a local variable of type `JSONObject` in the method `fromString`